src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -14,4 +29,21 @@
 public class WalletController {
 
     private final WalletService walletService;
+
+    @PutMapping
+    public ResponseEntity<WalletDTO> updateWallet(@RequestBody @Valid UpdateWalletDTO updateWalletDto) {
+
+        WalletDTO walletDTO = walletService.updateWallet(updateWalletDto);
+        HttpHeaders headers = new HttpHeaders();
+        headers.add("message", "You have successfully updated Wallet!");
+        return new ResponseEntity<>(walletDTO, headers, HttpStatus.OK);
+    }
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    Map<String, String> handleMethodArgumentNotValidException(MethodArgumentNotValidException ex) {
```

Comment:
Why this method is not inside .. i.e: `WalletExceptionHandler` class?

[**Almost** good implemented `WalletExceptionHandler`](
https://github.com/bycza-zagroda/track-expenses-app-backend/blob/9c42da340b00f8f9c2d380557014114007aed9b4/src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/WalletExceptionHandler.java)

Reply:
Nope. `GlobalExceptionHandler..` is to catch for example:

-     SQLException
-     SQLNonTransientException
-     SQLTransientException
-     SQLRecoverableException
-     etc..

Reply:
OK, added WalletExceptionHandler class.